easybuild/easyconfigs/b/btop/btop-1.3.2-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'btop'
+version = "1.3.2"
+
+homepage = 'https://github.com/aristocratos/btop'
+
+description = """Resource monitor that shows usage and stats for processor, memory, disks, network and processes."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+github_account = 'aristocratos'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['331d18488b1dc7f06cfa12cff909230816a24c57790ba3e8224b117e3f0ae03e']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.26.3'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/btop'],
```

Comment:
@Crivella Can we add a sanity check command like `btop --help` as well?

Reply:
Fixed with [08dcdae](https://github.com/easybuilders/easybuild-easyconfigs/pull/21373/commits/08dcdae217aaff65e2f69f2fa36875970848d58c)